content/en/docs/v3.5/_index.md

```diff
@@ -7,7 +7,7 @@ cascade:
   exclude_search: false
 linkTitle: *name
 simple_list: true
-weight: -350 # Weight for doc version vX.Y should be -XY0
+weight: -360 # Weight for doc version vX.Y should be -XY0
```

Comment:
this one should stay `-350` -- if it's the same as v3.6's  weight, then the ordering on the left hand menu defaults to alphabetical and v3.5 will come in a above v3.6

Reply:
@nate-double-u ah, I got mixed up with the files names. Good catch.